src/spacel/model/app.py

```diff
@@ -21,9 +21,17 @@ def __init__(self, orbit, params=None):
         self.instance_type = params.get('instance_type', 't2.nano')
         self.instance_min = params.get('instance_min', 1)
         self.instance_max = params.get('instance_max', 2)
-        self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = self._str2bool(params.get('loadbalancer', 'true'))
+
+        self.instance_scheme = params.get('instance_scheme', 'private')
+        self.instance_availability = params.get('instance_availability',
+                                                self.instance_scheme)
+
+        self.elb_scheme = params.get('elb_scheme', 'internet-facing')
+        self.elb_availability = params.get('elb_availability', self.elb_scheme)
```

Comment:
Do we need to store this twice? Looks like `self.elb_availability` is the only one used outside of the class.

I don't think we need to worry about backwards compat atm, "one way to do things" seems best.